lib/interactiveCli/set-app.test.js

```
@@ -194,26 +223,72 @@ describe('interactiveCli: set-app', function() {
       expect(serverless.service.tenant).to.equal('testinteractivecli');
       expect(serverless.service.app).to.equal('other-app');
     });
+  });
+
+  describe('Monitoring setup from CLI flags', () => {
+    const serviceConfigPath = join(awsLoggedInServicePath, 'serverless.yml');
+    let originalServiceConfig;
+    before(async () => (originalServiceConfig = await readFile(serviceConfigPath)));
+    afterEach(() => {
+      if (originalServiceConfig) return writeFile(serviceConfigPath, originalServiceConfig);
+      return null;
+    });
+
+    it('Should setup monitoring for chosen org and app', async () => {
+      configureInquirerStub(inquirer, {
+        confirm: { shouldEnableMonitoring: true },
+        list: { tenantName: 'invalid-testinteractivecli', appName: 'invalid-other-app' },
+      });
```

If this is test for typical (non erroneous-case) scenario, where for service which is not setup for monitoring, valid (against logged-in user) `--org` and `--app` are passed.

Then iirc we should expect no prompts, so no Inquirer stub should be configured (if it is it may hide errors in our setup)